docs/deep-dive/scaffolding/your-first-scaffold-plugin.md

```diff
@@ -0,0 +1,135 @@
+---
+id: your-first-scaffold-plugin
+title: Your first scaffold plugin
+sidebar_label: Your first scaffold plugin
+---
+
+This guide will help you create your own scaffolding plugins using our [Plugins system](/docs/deep-dive/plugins-crash-course).
+
+# Alright, how do I do it?
```

Comment:
Maybe change the title to something more formal:
Like -> "How to start" or "Where to start"

Reply:
yep